weblate_web/templates/snippets/payment.html

```diff
@@ -1,33 +1,36 @@
 {% load i18n %}
 {% load humanize %}
+{% load date_diff %}
+{% load static %}
+
+
 <div class="payment-listing">
-    <div class="float-right">
-        {{ payment.get_amount_display|intcomma }} {{ payment.get_currency_display }}
-    </div>
-    {{ payment.description }}<br />
+    <div class="float-right payment-font-size">
+        {{ payment.get_amount_display|intcomma }} {{ payment.get_currency_display }}<span>{% if payment.invoice %}
```

Comment:
Please move the condition to next line to make the code easier to read:
```suggestion
        {{ payment.get_amount_display|intcomma }} {{ payment.get_currency_display }}<span>
        {% if payment.invoice %}
```

Reply:
Yes, for sure. I have fixed that.